src/app/public/modules/inline-form/types/inline-form-type.ts

```diff
@@ -0,0 +1,7 @@
+export enum SkyInlineFormType {
```

Comment:
Something about the name of this enum feels off? It seems to be describing the button layout; what if it was more descriptive and named `SkyInlineFormButtonLayout`? This enum governs the layout and appearance of specific buttons?

Reply:
For example, what if in the future we wanted a different "type" of form, but the same button layout?